<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>942</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1086</width>
    <height>942</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(0, 0, 0), stop:1 rgb(255, 255, 255));
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string>background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(30, 30, 30, 255), stop:1 rgba(70, 70, 70, 255));</string>
   </property>
   <widget class="QLabel" name="music_title">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>15</y>
      <width>1061</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(220, 220, 220), stop:1 rgb(180, 180, 180));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="performance_toolbar" native="true">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>70</y>
      <width>1061</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(220, 220, 220), stop:1 rgb(180, 180, 180));</string>
    </property>
    <widget class="QPushButton" name="record_btn">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>background-color: rgba(145, 145, 145, 200);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>../../../Downloads/Icons/record.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>500</width>
       <height>105</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="bpm_slider">
     <property name="geometry">
      <rect>
       <x>601</x>
       <y>41</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>background-color: rgba(145, 145, 145, 200);</string>
     </property>
     <property name="minimum">
      <number>40</number>
     </property>
     <property name="maximum">
      <number>240</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_bpm">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>background-color: rgba(145, 145, 145, 200); border-color: rgb(88, 88, 88);</string>
     </property>
     <property name="text">
      <string>BPM Adjuster</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
